docs/pull_requests.md

```diff
@@ -0,0 +1,30 @@
+# Pull Requests
+
+## For creating a PR
+Push your branch locally to the repo
```

Comment:
"Push your local branch to the repo" might be clearer. In this case local describes the branch, not the push.